pkg/handler/handler.go

```diff
@@ -277,6 +292,39 @@ func New(l *zap.Logger, baseDir string) (*mux.Router, error) {
 	return router, nil
 }
 
+func getSingleContainerPodContainerName(apiURL, namespace, name string) (string, error) {
+	resp, err := http.Get(fmt.Sprintf("%s/api/v1/namespaces/%s/pods/%s", apiURL, namespace, name))
+	if err != nil {
+		return "", fmt.Errorf("failed to get pod: %w", err)
+	}
+
+	podBytes, err := io.ReadAll(resp.Body)
+	if err != nil {
+		return "", fmt.Errorf("failed to read response body: %w", err)
+	}
+
+	var pod corev1.Pod
+	if err := yaml.Unmarshal(podBytes, &pod); err != nil {
+		return "", fmt.Errorf("unmarshal failed: %w", err)
+	}
+
+	if len(pod.Spec.Containers) == 0 {
+		return "", fmt.Errorf("invalid pod zero containers")
```

Comment:
Please don't use `fmt` for static strings

```suggestion
		return "", errors.New("invalid pod zero containers")
```